actors/builtin/multisig/multisig_test.go

```diff
@@ -979,8 +980,8 @@ func (h *msActorHarness) assertTransactions(rt *mock.Runtime, expected ...multis
 	require.Equal(h.t, len(expected), len(keys))
 	for i, k := range keys {
 		var actual multisig.MultiSigTransaction
-		found, err := txns.Get(asKey(k), &actual)
-		require.NoError(h.t, err)
+		found, err_ := txns.Get(asKey(k), &actual)
+		require.NoError(h.t, err_)
```

Comment:
Avoiding the shadow here feels a bit extreme, is this necessary?

Reply:
I would like to reach the point of enabling it as a linter rule eventually